pom.xml

```diff
@@ -25,6 +25,32 @@
         <module>javaparser-testing</module>
     </modules>
 
+  <profiles>
+    <profile>
+      <id>default-tools.jar</id>
+      <activation>
+        <property>
+          <name>java.vendor</name>
+          <value>Oracle Corporation</value>
+        </property>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>com.sun</groupId>
+          <artifactId>tools</artifactId>
+          <version>1.7</version>
```

Comment:
I think we are still supporting JDK 6 and I am not sure if we can insert this dependency